cuebot/src/main/resources/opencue.properties

```diff
@@ -25,10 +25,14 @@ grpc.rqd_task_deadline=10
 # Set to a boolean value. See com/imageworks/spcue/services/JmsMover.java.
 messaging.enabled=false
 
-# Root directory for which logs will be stored. See com/imageworks/spcue/util/JobLogUtil.java.
+# Default root directory for which logs will be stored no other OS is defined.
+# See com/imageworks/spcue/util/JobLogUtil.java.
 # Override this via environment variable (CUE_FRAME_LOG_DIR) or command line flag
-# (--log.frame-log-root). Command line flag will be preferred if both are provided.
-log.frame-log-root=${CUE_FRAME_LOG_DIR:/shots}
+# (--log.frame-log-root.default_os). Command line flag will be preferred if both are provided.
+log.frame-log-root.default_os=${CUE_FRAME_LOG_DIR:/shots}
+# To set up root directories for other OS create new environment
+# variable as `log.frame-log-root.[OS] where OS relates to str_os on the job table
```

Comment:
Can you add commented-out Windows and Linux examples?

Reply:
Just added  examples:
```
# For example:
# log.frame-log-root.Windows=arbitraryWindowsDirectory
# log.frame-log-root.linux=arbitraryLinuxDirectory
```

Reply:
Could you expand those examples to include a (fake) path?

For example if I want to define a Windows log root, do I use forward slashes, backslashes, or is either ok? An example to work from would make it easier to know without having to figure it out for myself.